ansible/roles/stack-sunbird/templates/ml-core-service.env

```diff
@@ -54,7 +54,8 @@ GOTENBERG_SERVICE_URL={{ml_gotenberg_url | default("http://gotenberg:3000")}}
 #   AZURE_ACCOUNT_KEY=xx123supersecretpassword
 #   AZURE_STORAGE_CONTAINER=slassessmentscontainer
 
-{{ml_core_cloud_config}}
+{{ml_cloud_config}}
```

Comment:
What is this? How will we define this in private? where is the default for this? is the a dictionary, list, string, integer etc?